src/utils/CatalystClientBuilder.ts

```diff
@@ -13,17 +14,20 @@ export async function clientConnectedToCatalystIn(
     preKnownServers: { network },
     origin
   })
+
   let list: string[]
   if (noContractList) {
     list = noContractList
   } else {
     console.warn('Falling back to the smart contract to get an updated list of active servers')
     list = await getApprovedListFromContract(network)
   }
+
   const shuffled = shuffleArray(list)
```

Comment:
Weird thought, and I wanted to see your opinion. Do you think there is a good/non-expensive way to prioritize up to date servers and de-prioritize ones that have not been updated for a while?

Might make sense to tackle it in another issue though

Reply:
The list is 12 peers long, so any logic will be non expensive, but I would do it in another PR